lib/helpers/function.ts

```diff
@@ -0,0 +1,58 @@
+export const noop = () => {};
+export const identity = (value: any) => value;
+
+export const debounce = (func: Function, wait: number, isImmediate?: boolean) => {
+    let timeoutId: ReturnType<typeof setTimeout> | undefined;
+
+    return function executedFunction(this: any, ...args: any[]) {
+        const context = this;
```

Comment:
I'm getting:

```
Unexpected aliasing of 'this' to local variable.eslint(@typescript-eslint/no-this-alias)
```

I don't really see a reason to alias it anyway, you could just use `this`, instead of `context` everywhere, since arrow functions don't create a scope.